app/subscriptions/models.py

```diff
@@ -0,0 +1,76 @@
+from django.db import models
+
+
+class Category(models.Model):
```

Comment:
Давай наследоваться от базовой модели из cors
```suggestion
class Category(BaseModel):
```